src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,166 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String INDEX_OUT_OF_BOUNDS_EXCEPTION
+            = "Invalid index: ";
+    private static final String SIZE = ", Size: ";
```

Comment:
String format would be more handy to use instead of this

Reply:
Understood, will stick to it in the future